`make release` failure

## Bug Report

<!-- provide description here -->
`make release` fails with the following error
```
GOPRIVATE="github.com/vmware-tanzu-private" go install -ldflags "-s -w -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildDate=$(date -u +"%Y-%m-%d")' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildSHA=$(git describe --match= --always)' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildVersion=v1.3.0'" ./cmd/cli/tanzu
+ popd
/tmp/tce-release
+ rm -rf /tmp/tce-release/tanzu-cli-tkg-plugins
+ set +x
Cloning into 'tanzu-cli-tkg-plugins'...
remote: Enumerating objects: 36, done.
remote: Counting objects: 100% (36/36), done.
remote: Compressing objects: 100% (31/31), done.
remote: Total 36 (delta 4), reused 17 (delta 0), pack-reused 0
Receiving objects: 100% (36/36), 69.87 KiB | 2.91 MiB/s, done.
Resolving deltas: 100% (4/4), done.
+ pushd /tmp/tce-release/tanzu-cli-tkg-plugins
/tmp/tce-release/tanzu-cli-tkg-plugins /tmp/tce-release
+ git reset --hard
HEAD is now at f7aaa1a restore cluster-api redirect
+ go mod edit --replace github.com/vmware-tanzu-private/tkg-cli=../tkg-cli
+ go mod edit --replace github.com/vmware-tanzu-private/tkg-providers=../tkg-providers
+ go mod edit --replace github.com/vmware-tanzu-private/core=../core
+ sed -i.bak -e 's/ --dirty//g' ./Makefile
+ rm ./Makefile.bak
+ sed -i.bak -e 's/$(shell git describe --tags --abbrev=0 2>$(NUL))/v1.3.0/g' ./Makefile
+ rm ./Makefile.bak
+ sed -i.bak -e 's/tanzu builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path .\/cmd\/plugin/$(GO) run github.com\/vmware-tanzu-private\/core\/cmd\/cli\/plugin-admin\/builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path .\/cmd\/plugin --artifacts $(ARTIFACTS_DIR)/g' ./Makefile
+ rm ./Makefile.bak
+ sed -i.bak -e 's/tanzu plugin install all --local $(ARTIFACTS_DIR)/TANZU_CLI_NO_INIT=true $(GO) run -ldflags "$(LD_FLAGS)" github.com\/vmware-tanzu-private\/core\/cmd\/cli\/tanzu plugin install all --local $(ARTIFACTS_DIR)/g' ./Makefile
+ rm ./Makefile.bak
+ make build
GOPRIVATE="github.com/vmware-tanzu-private" go mod download
GOPRIVATE="github.com/vmware-tanzu-private" go run github.com/vmware-tanzu-private/core/cmd/cli/plugin-admin/builder cli compile --version v1.3.0 --ldflags "-s -w -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildDate=$(date -u +"%Y-%m-%d")' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildSHA=$(git rev-parse --short HEAD)' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildVersion=v1.3.0'" --path ./cmd/plugin --artifacts ./artifacts
ℹ  building local repository at ././artifacts
-------
ℹ  building plugin at path "cmd/plugin/managementcluster"
✖  error: exit status 2
✖  output: # github.com/vmware-tanzu-private/tkg-cli/pkg/client
../tkg-cli/pkg/client/delete_standalone.go:191:20: undefined: "sigs.k8s.io/cluster-api/cmd/clusterctl/client".RestoreOptions
../tkg-cli/pkg/client/init_standalone.go:229:17: undefined: "sigs.k8s.io/cluster-api/cmd/clusterctl/client".SaveOptions

exit status 1
make[1]: *** [build] Error 1
+ make install-cli-plugins
TANZU_CLI_NO_INIT=true GOPRIVATE="github.com/vmware-tanzu-private" go run -ldflags "-s -w -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildDate=$(date -u +"%Y-%m-%d")' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildSHA=$(git rev-parse --short HEAD)' -X 'github.com/vmware-tanzu-private/core/pkg/v1/cli.BuildVersion=v1.3.0'" github.com/vmware-tanzu-private/core/cmd/cli/tanzu plugin install all --local ./artifacts -u
Error: could not find manifest.yaml file: open artifacts/manifest.yaml: no such file or directory
✖  could not find manifest.yaml file: open artifacts/manifest.yaml: no such file or directory 
exit status 1
make[1]: *** [install-cli-plugins] Error 1
+ popd
```
## Expected Behavior
`make release` should pass successfully 
<!-- provide what you expected to happen here -->

## Steps to Reproduce the Bug
Run `make release`
<!-- provide steps to reproduce here -->

## Environment Details

* Build version (`tanzu version`):  
TCE version v0.6.0-dev.1
```
version: v1.3.0
buildDate: 2021-05-29
sha: fa7b282
```
* Operating System (client): Darwin

--- a/Makefile
+++ b/Makefile
@@ -213,7 +213,7 @@ build-cli: install-cli
 
 .PHONY: install-cli
 install-cli:
-	TANZU_CORE_REPO_BRANCH="tce-v1.3.0" TKG_CLI_REPO_BRANCH="tce-v1.3.0-saui" TKG_PROVIDERS_REPO_BRANCH="tce-v1.3.0" TANZU_TKG_CLI_PLUGINS_REPO_BRANCH="tce-v1.3.0" BUILD_VERSION=${CORE_BUILD_VERSION} hack/build-tanzu.sh
+	TANZU_CORE_REPO_BRANCH="tce-v1.3.0" TKG_CLI_REPO_BRANCH="tce-v1.3.0-saui" CLUSTER_API_REPO_BRANCH="tce-v0.3.14" TKG_PROVIDERS_REPO_BRANCH="tce-v1.3.0" TANZU_TKG_CLI_PLUGINS_REPO_BRANCH="tce-v1.3.0" BUILD_VERSION=${CORE_BUILD_VERSION} hack/build-tanzu.sh
 
 .PHONY: clean-core
 clean-core: clean-cli-metadata

--- a/hack/build-tanzu.sh
+++ b/hack/build-tanzu.sh
@@ -27,6 +27,10 @@ TKG_CLI_REPO_BRANCH=${TKG_CLI_REPO_BRANCH:-$BUILD_VERSION}
 if [[ "${TKG_CLI_REPO_BRANCH}" != "${BUILD_VERSION}" ]]; then
     echo "**************** WARNING - TKG_CLI_REPO_BRANCH = ${TKG_CLI_REPO_BRANCH} ****************"
 fi
+CLUSTER_API_REPO_BRANCH=${CLUSTER_API_REPO_BRANCH:-$BUILD_VERSION}
+if [[ "${CLUSTER_API_REPO_BRANCH}" != "${BUILD_VERSION}" ]]; then
+    echo "**************** WARNING - CLUSTER_API_REPO_BRANCH = ${CLUSTER_API_REPO_BRANCH} ****************"
+fi
 TANZU_CORE_REPO_BRANCH=${TANZU_CORE_REPO_BRANCH:-$BUILD_VERSION}
 if [[ "${TANZU_CORE_REPO_BRANCH}" != "${BUILD_VERSION}" ]]; then
     echo "**************** WARNING - TANZU_CORE_REPO_BRANCH = ${TANZU_CORE_REPO_BRANCH} ****************"
@@ -52,14 +56,29 @@ pushd "${ROOT_REPO_DIR}/tkg-cli" || exit 1
 git reset --hard
 popd || exit 1
 
+rm -rf "${ROOT_REPO_DIR}/cluster-api"
+set +x
+git clone --depth 1 --branch "${CLUSTER_API_REPO_BRANCH}" "https://git:${GH_ACCESS_TOKEN}@github.com/vmware-tanzu/cluster-api.git" "cluster-api"
+set -x
+pushd "${ROOT_REPO_DIR}/cluster-api" || exit 1
+git reset --hard
+popd || exit 1
+
 rm -rf "${ROOT_REPO_DIR}/core"
 mv -f "${HOME}/.tanzu" "${HOME}/.tanzu-$(date +"%Y-%m-%d_%H:%M")"
 set +x
 git clone --depth 1 --branch "${TANZU_CORE_REPO_BRANCH}" "https://git:${GH_ACCESS_TOKEN}@github.com/vmware-tanzu-private/core.git" "core"
 set -x
+
+pushd "${ROOT_REPO_DIR}/tkg-cli" || exit 1
+git reset --hard
+go mod edit --replace sigs.k8s.io/cluster-api=../cluster-api
+popd || exit 1
+
 pushd "${ROOT_REPO_DIR}/core" || exit 1
 git reset --hard
 go mod edit --replace github.com/vmware-tanzu-private/tkg-cli=../tkg-cli
+go mod edit --replace sigs.k8s.io/cluster-api=../cluster-api
 go mod edit --replace github.com/vmware-tanzu-private/tkg-providers=../tkg-providers
 sed -i.bak -e "s/ --dirty//g" ./Makefile && rm ./Makefile.bak
 sed -i.bak -e "s/\$(shell git describe --tags --abbrev=0 2>\$(NUL))/${BUILD_VERSION}/g" ./Makefile && rm ./Makefile.bak
@@ -75,6 +94,7 @@ git reset --hard
 go mod edit --replace github.com/vmware-tanzu-private/tkg-cli=../tkg-cli
 go mod edit --replace github.com/vmware-tanzu-private/tkg-providers=../tkg-providers
 go mod edit --replace github.com/vmware-tanzu-private/core=../core
+go mod edit --replace sigs.k8s.io/cluster-api=../cluster-api
 sed -i.bak -e "s/ --dirty//g" ./Makefile && rm ./Makefile.bak
 sed -i.bak -e "s/\$(shell git describe --tags --abbrev=0 2>\$(NUL))/${BUILD_VERSION}/g" ./Makefile && rm ./Makefile.bak
 sed -i.bak -e "s/tanzu builder cli compile --version \$(BUILD_VERSION) --ldflags \"\$(LD_FLAGS)\" --path .\/cmd\/plugin/\$(GO) run github.com\/vmware-tanzu-private\/core\/cmd\/cli\/plugin-admin\/builder cli compile --version \$(BUILD_VERSION) --ldflags \"\$(LD_FLAGS)\" --path .\/cmd\/plugin --artifacts \$(ARTIFACTS_DIR)/g" ./Makefile && rm ./Makefile.bak

